.travis.yml

```diff
@@ -31,24 +35,25 @@ addons:
     - cmake
     - cmake-data
 before_install:
-  - $CC --version
+  - $CC --version 
```

Comment:
There is a trailing space at this line.  Could you remove it?

Reply:
Done !